cmd/docker/main.go

```diff
@@ -106,6 +106,8 @@ func handleFlags() (*CliOptions, bool) {
 	portLayerAddr := flag.String("port-layer-addr", "127.0.0.1", "Port layer server address")
 	portLayerPort := flag.Uint("port-layer-port", 9001, "Port Layer server port")
 
+	debug := flag.Bool("debug", false, "Enable debuglevel logging")
```

Comment:
We should start pulling this data from vchConfig. There are two sources, one is appliance wide, and the other is session specific.
```
vchConfig.ExecutorConfig.Diagnostics.DebugLevel
vchConfig.ExecutorConfig.Sessions["docker-personality"].Diagnostics.DebugLevel
```